CHANGELOG.md

```diff
@@ -25,12 +25,11 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 ### Dependencies
 - Bump `org.apache.logging.log4j:log4j-core` from 2.18.0 to 2.20.0 ([#6490](https://github.com/opensearch-project/OpenSearch/pull/6490))
 - Bump `com.azure:azure-storage-common` from 12.19.3 to 12.20.0 ([#6492](https://github.com/opensearch-project/OpenSearch/pull/6492)
-- Bump `com.azure:azure-storage-blob` from 12.15.0 to 12.20.0 ([#6921](https://github.com/opensearch-project/OpenSearch/pull/6921)
```

Comment:
Deleted Line 28 because Line 41 saying the same, and removing this line makes it identical with `main` branch (https://github.com/opensearch-project/OpenSearch/blob/53b128fc10f4b2ee5c95b55000dbfa89f110a661/CHANGELOG.md?plain=1#L116)